src/libraries/System.Runtime.Serialization.Schema/tests/System.Runtime.Serialization.Schema.Tests.csproj

```
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\Runtime\Serialization\Schema\Import.cs" />
+    <Compile Include="System\Runtime\Serialization\Schema\Contracts.cs" />
+    <Compile Include="System\Runtime\Serialization\Schema\Export.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Expected.ImportedType.cs">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
```

nit: please use the less verbose syntax for item metadata declaration:

```suggestion
    <None Include="Expected.ImportedType.cs"
          CopyToOutputDirectory="Always" />
```